### ISSUE TITLE
Windows separator for Assets.absoluteFilePath

### ISSUE THREAD

**Issue by PrestonGiorgianni** [meteor/meteor#11312]
## Meteor version
Meteor 1.10.2 (not fixed in newer version when tested)

## Operating System
Windows 10 20H2

## Expected behavior
```javascript
const path = Assets.absoluteFilePath('test_files\\small_test_2.txt');
console.log(path);
```
This code should log the path to the directory where meteor is storing the assets. (i.e. `C:\Users\User\AppData\Local\Temp\meteor-test-run1n0c2i7.5qdo\.meteor\local\build\programs\server\assets\app\test_files\small_test_2.txt`

## Actual behavior

An error is thrown when trying to do this.
```
Error: Unknown asset: test_files\small_test_1.txt
     at Object.absoluteFilePath (C:\Users\User\AppData\Local\Temp\meteor-test-run1n0c2i7.5qdo\.meteor\local\build\programs\server\boot.js:357:19)
     at tests/endToEnd-tests.js:35:23
...
```

## steps to reproduce
In progress for making a small repo that can reproduce this issue

## Proposed solution

On this [line](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/tools/static-assets/server/boot.js#L364) the fileInfo.assets has all linux based file paths (i.e. part1/part2) and when passing it a windows file path of (i.e part1\part2) the key does not match the entry in the array. On this [line](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/tools/static-assets/server/boot.js#L368) they convert the path to look like a linux path, after they have already checked if it exists in the assets array (aka. too late so the error was thrown before it can translate)

The solution would be to move the line 368 to 363. I modified my local copy of meteor to see that this resolved the issue.

**Comment by filipenevola** [comment]
Hi @PrestonGiorgianni, thank you for your report.

It seems like you are pretty close to have a solution for your issue, right?

You can submit your PR when you have the code fixed and so we can include it in the next Meteor version.

Thanks.

**Comment by filipenevola** [comment]
It is going to be fixed on Meteor 2.2: PR #11346
